### ISSUE TITLE
[Enhancement] Bump baseOS to SLE Micro for Rancher 5.3

### ISSUE THREAD

**Issue by bk201** [harvester/harvester#3279]
**Describe the bug**

We need to bump the base to `suse/sle-micro-rancher/5.3:latest` in https://build.opensuse.org/package/view_file/isv:Rancher:Harvester:BaseOS/baseos/Dockerfile?expand=1

**Additional context**
5.3 will be the supported upstream version.

**Comment by Vicente-Cheng** [comment]
upgrade blocker: https://github.com/harvester/harvester/issues/3444
Somehow the behavior has changed after we introduce SLE Micro for Rancher 5.3

**Comment by TachunLin** [comment]
### Test Plan
1. Fresh installation Harvester 
    - check the following package is contained
      1. k9s v0.26.7-150400.6.3
      2. wicked, wicked-service v0.6.70-150400.3.3.1
    - Check basic operation through terraform
    
2. Upgrade path with this ISO (Until #3444 merged)
    - check the following package is contained
      1. k9s v0.26.7-150400.6.3
      2. wicked, wicked-service v0.6.70-150400.3.3.1
    - Check basic operation through terraform

**Comment by TachunLin** [comment]
### Test Plan 1 Result

* - [x] Fresh install Harvester `master-213d4af7-head` 
* - [x] Fresh install Harvester `v1.1.2-rc2` 

* - [x] Check OS version

  ```
  harvester-t4mvc:~ # cat /etc/os-release
  NAME="SLE Micro"
  VERSION="5.3"
  VERSION_ID="5.3"
  PRETTY_NAME="Harvester v1.1.2-rc1"
  ID="sle-micro-rancher"
  ID_LIKE="suse"
  ANSI_COLOR="0;32"
  CPE_NAME="cpe:/o:suse:sle-micro-rancher:5.3"
  VARIANT="Harvester"
  VARIANT_ID="Harvester-20230209"
  GRUB_ENTRY_NAME="Harvester v1.1.2-rc1"
  ```

* - [x] Check K9s version is v0.26.7-150400.6.3

  ```
  harvester-t4mvc:~ # sudo zypper search -s k9s
  Loading repository data...
  Warning: No repositories defined. Operating only with the installed resolvables. Nothing can be installed.
  Reading installed packages...
  
  S  | Name | Type    | Version           | Arch   | Repository
  ---+------+---------+-------------------+--------+------------------
  i+ | k9s  | package | 0.26.7-150400.6.3 | x86_64 | (System Packages)
  
  ```

* - [x] Check wicked, wicked-service version is v0.6.70-150400.3.3.1
  ```
  harvester-t4mvc:~ # sudo zypper search -s wicked
  Loading repository data...
  Warning: No repositories defined. Operating only with the installed resolvables. Nothing can be installed.
  Reading installed packages...
  
  S  | Name           | Type    | Version             | Arch   | Repository
  ---+----------------+---------+---------------------+--------+------------------
  i+ | wicked         | package | 0.6.70-150400.3.3.1 | x86_64 | (System Packages)
  i  | wicked-service | package | 0.6.70-150400.3.3.1 | x86_64 | (System Packages)
  ```


* Use terraform provider for some basic operations 
   - [x] image
   - [x] volume
   - [x] virtual machines 

* Check the network basic operations 
   - [x] cluster network
   - [x] vlanconfig
   - [x] netowork

**Comment by TachunLin** [comment]
Move to ready for test stage first since the test plan 2 of upgrade path requires https://github.com/harvester/harvester/issues/3444 to be merged.

**Comment by TachunLin** [comment]
### Test Plan 2 Result

* - [x] Upgrade from v1.1.2-master to the same master version correctly 

* - [x] Check OS version
  ```
  harv3144:~ # cat /etc/os-release
  NAME="SLE Micro"
  VERSION="5.3"
  VERSION_ID="5.3"
  PRETTY_NAME="Harvester master"
  ID="sle-micro-rancher"
  ID_LIKE="suse"
  ANSI_COLOR="0;32"
  CPE_NAME="cpe:/o:suse:sle-micro-rancher:5.3"
  VARIANT="Harvester"
  VARIANT_ID="Harvester-20230209"
  GRUB_ENTRY_NAME="Harvester master"
  
  ```
* - [x] Check K9s version is `v0.26.7-150400.6.3`
  ```
  harv3144:~ # sudo zypper search -s k9s
  Loading repository data...
  Warning: No repositories defined. Operating only with the installed resolvables. Nothing can be installed.
  Reading installed packages...
  
  S  | Name | Type    | Version           | Arch   | Repository
  ---+------+---------+-------------------+--------+------------------
  i+ | k9s  | package | 0.26.7-150400.6.3 | x86_64 | (System Packages)
  ```

* - [x] Check wicked, wicked-service version is `v0.6.70-150400.3.3.1`

  ```
  harv3144:~ # sudo zypper search -s wicked
  Loading repository data...
  Warning: No repositories defined. Operating only with the installed resolvables. Nothing can be installed.
  Reading installed packages...
  
  S  | Name           | Type    | Version             | Arch   | Repository
  ---+----------------+---------+---------------------+--------+------------------
  i+ | wicked         | package | 0.6.70-150400.3.3.1 | x86_64 | (System Packages)
  i  | wicked-service | package | 0.6.70-150400.3.3.1 | x86_64 | (System Packages)
  ```

* Use terraform provider for some basic operations 
   - [x] image
   - [x] volume
   - [x] virtual machines 

* Check the network basic operations 
   - [x] cluster network
   - [x] vlanconfig
   - [x] netowork

**Comment by TachunLin** [comment]
Verified **fixed** on `v1.1.2-rc1` and `master-89b3108b-head`. Close this issue.

### Result
Verified can pass the following test cases for fresh install and upgrade path 

* Check the os version bump to `sle-micro-rancher:5.3`
* Check the following package is contained

    - k9s v0.26.7-150400.6.3
    - wicked, wicked-service v0.6.70-150400.3.3.1

* Check basic operation through terraform 

Fresh install result: https://github.com/harvester/harvester/issues/3279#issuecomment-1429292251
Upgrade path result: https://github.com/harvester/harvester/issues/3279#issuecomment-1437912622

### Test Information
* Test Environment: 4 nodes Harvester bare machines and 1 node local kvm machine
* Harvester version: `v1.1.2-rc1` and `master-89b3108b-head`

### Verify Steps
1. Fresh install or upgrade the same version to v1.1.2 master 
2. Check the os version 
3. Check the k9s version
4. Check the wicked, wicked-service version
